### ISSUE TITLE
FragmentDescriptors 

### ISSUE THREAD

**Issue by nicogam** [rdkit/rdkit#3552]
Hi everyone! I would ask to you if there is a way to modify the "FragmentDescriptors.csv", as I have to add a new fragment not included in this list. (any other suggestions to do so are welcome!)

Thanks to everyone in advance.

PS: i'm using rdkit 2020.03.2 version on python3.7.

Nico

**Comment by bp-kelley** [comment]
Do you have the ability to edit the file in $RDBASE/Data/FragmentDescriptors.csv ?

RDBASE varies on platform to platform, you can find it via:

```
from rdkit import RDConfig
print(RDConfig.RDDataDir)
```

**Comment by bp-kelley** [comment]
It doesn't look like there is an easy way to reload the fragments, but this can be fixed.

**Comment by nicogam** [comment]
> Do you have the ability to edit the file in $RDBASE/Data/FragmentDescriptors.csv ?
> 
> RDBASE varies on platform to platform, you can find it via:
> 
> ```
> from rdkit import RDConfig
> print(RDConfig.RDDataDir)
> ```

First of all, thanks for the answer!
As for what you said, I tried to edit the file FragmentDescriptors.csv in two different ways: 
1) I tried to add a new line with my new fragment code and SMARTS, but it didn't work.
2) I tried (cheating) to modify only the SMARTS code of another fragment already existing, and in this work, but I think that it isn't the right way! :D 
So I'll try to figure out somehow!
